src/Bridge/Symfony/Bundle/Resources/function.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+// Forward compatibility layer for the "service()" method.
+// To be removed along with support for symfony/dependency-injection <5.1.
+
+if (!\function_exists(__NAMESPACE__.'\service')) {
+    eval('
```

Comment:
No way 😁

Either we bump the minimal version of dependency-injection to 5.1, or we keep XML for now and merge this PR in 3.0, but it's too ugly for a very limited benefit (if any) IMHO.

Reply:
Fair enough :) Let's bump Symfony to 5.1+ and merge this in 3.x then